doc/build/changelog/unreleased_21/10635.rst

```diff
@@ -0,0 +1,6 @@
+.. change::
+    :tags: typing
+    :tickets: 10635
+
+    Add pep-646 integration and deprecate the old tuple-workaround.
+    Patch provided by Yurii Karabas.
```

Comment:
**Michael Bayer** (zzzeek) wrote:

I will add a full migration notes section

note that people's existing applications will break with this change if they have any kind of explicit typing for the tuple thing

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
> I will add a full migration notes section

So you will going to do it? No need to do anything from my side?

Reply:
I'll do the migration notes, sure, since I already wrote the huge explanation of Tuple for 2.0, this is just the same story I want to continue from the perspective I've had on this.

I didnt go through the rest of the files but aboslutely we can finish up here at this point

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986